### DISCUSSION TITLE
Could the Chinese video record be uploaded to Youtube?

### DISCUSSION THREAD

**Discussion by ThomsonTan** [mlc-ai/mlc-en#8] [General]
For now the Chinese video record is only uploaded bi bilibili.com, upload it to Youtube may help anyone who live oversea but prefer the Chinese version.

**Comment by Hzfengsy** [comment]
cc @tqchen

**Comment by tqchen** [comment]
Thanks for the suggestion, to keep things simple and consistent, we decided that to have one upload location for each type of video for now
